dashboard/src/components/ErrorAlert/UnexpectedErrorAlert.tsx

```diff
@@ -55,7 +55,7 @@ class UnexpectedErrorPage extends React.Component<IUnexpectedErrorPage> {
     // typescript does not support react's defaultProps and we are running it in strictNullChecks mode.
     // Newer versions of it seems to support it https://github.com/Microsoft/TypeScript/wiki/Roadmap#30-july-2018
     return (
-      <div className="alert alert-error margin-t-bigger">
+      <div className="alert alert-error margin-bigger">
```

Comment:
The only concern I'd have about this is that other alerts don't have the same change, it could make things look inconsistent?